### DISCUSSION TITLE
"xxx" language in Comments tag

### DISCUSSION THREAD

**Discussion by sergioboni72** [Marekkon5/onetagger#197] [Q&A] [answered]
Hi, is there a way to set the program to write an "xxx" in language section of COMM tags? I noted that if it set to eng, it does not appear to users that have Windows in other language (my case). Changing to xxx shows it to everyone.

**Comment by Marekkon5** [comment, marked as answer]
Added in latest commit. You can get the binary from the Actions tab to test it out.
